src/content/docs/project/runs/managing-runs.md

```diff
@@ -78,12 +78,41 @@ To put Manual Run to a RunGroup you need to open it and click on Add Manual Run
 
 ![put Manual Run](./images/10.png)
 
-To put an Automated Run to a RunGroup you need to execute your tests with Testomat.io providing TESTOMATIO_RUNGROUP_TITLE="Build ${BUILD_ID}". 
+To put an Automated Run to a RunGroup you need to execute your tests with Testomat.io providing TESTOMATIO_RUNGROUP_TITLE="Build ${BUILD_ID}".
 
 Now you can see Test Runs in a created RunGroup
 
 ![Test Runs in a created RunGroup](./images/11.png)
 
+### How to Customize Your Runs List View
+
+When working with test runs in RunGroup, you can adjust the table layout to fit your needs. Instead of using the default view, you can customize the table by following these simple steps:
+
+1. Click Runs button in the sidebar
+2. Select RunGroup
+3. Click the Custom view button
+   ￼
+
+Let's check how a table with a custom view looks like.
+
+4. Click the Settings icon to access Runs List Settings.
+
+![Alt text](<images/CleanShot 2025-02-12 at 18.02.20@2x.png>)
+
+Then, you can customize the table layout:
+
+5. Select or deselect columns to show only the data you need
+6. Set the column width (px) if needed to improve readability
+7. Click Save button to apply your changes
+
+![Alt text](<images/CleanShot 2025-02-12 at 18.13.09@2x.png>)
```

Comment:
Better to avoid Alt text

Either to add a real description or remove Alt text phrase:

```
![Saving changes](./images/CleanShot 2025-02-12 at 18.13.09@2x.png>)
```